lib/views/teach_word_view.dart

```diff
@@ -143,11 +175,7 @@ class _TeachWordViewState extends State<TeachWordView>
                                     iconSize: 35,
```

Comment:
Align to the 2-space indent style with previous codes. Remove additional indents from 4 to 2.